apps/api/src/app/plugins/orm-analytics.ts

```diff
@@ -11,32 +11,37 @@ export default fp(async function (fastify: FastifyInstance) {
     database: fastify.config.COW_ANALYTICS_DATABASE_NAME,
     username: fastify.config.COW_ANALYTICS_DATABASE_USERNAME,
     password: fastify.config.COW_ANALYTICS_DATABASE_PASSWORD,
-  }
+  };
 
-  const dbParamsAreInvalid = Object.values(dbParams).some((v) => Number.isNaN(v) || v === undefined);
+  const dbParamsAreInvalid = Object.values(dbParams).some(
+    (v) => Number.isNaN(v) || v === undefined
+  );
 
   if (dbParamsAreInvalid) {
-    console.error('Invalid CoW Analytics database parameters, please check COW_ANALYTICS_* env vars');
-    return
+    console.error(
+      'Invalid CoW Analytics database parameters, please check COW_ANALYTICS_* env vars'
+    );
+    return;
   }
 
   fastify.register(typeORMPlugin, {
     ...dbParams,
+    namespace: 'analytics',
```

Comment:
namespacing helps to differenciate between the different `typeorm` instances. This one is for the analytics.